docs/pageobjects.md

```diff
@@ -3,15 +3,14 @@
 UI of your web application has interaction areas which can be shared across different tests.
 To avoid code duplication you can put common locators and methods into one place.
 
-## PageObjects
+## PageObject Generation
```

Comment:
better to keep it named PageObjects
if you think this part lacks the description or examples using pageobject, please add it